barretenberg/cpp/src/barretenberg/honk/proof_system/permutation_library.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 #include "barretenberg/common/ref_vector.hpp"
 #include "barretenberg/polynomials/polynomial.hpp"
+#include "barretenberg/relations/relation_parameters.hpp"
```

Comment:
Why did you add this header?

Reply:
Yeah I just double checked and it is needed. I guess it was an implicit dependency that was satisfied through one of the deleted files? This is defined in a header and the error happens when build relation_objects, so it's a little hard to tell the cause.